_release/bat/storage_bat/ceph_test.go

```diff
@@ -105,3 +128,21 @@ func TestCopyBlockDevice(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestMain(m *testing.M) {
+	var config configuration
+	data, err := ioutil.ReadFile("/etc/ciao/configuration.yaml")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error reading config file: %v", err)
+	} else {
+		err = yaml.Unmarshal(data, &config)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error parsing config file: %v", err)
+		}
+	}
+
+	driver.ID = config.Configure.Storage.CephID
```

Comment:
I would only initialise driver.ID if yaml.Unmarshal succeeds.  If it fails, the contents of driver.ID are essentially undefined so we could end up running the tests.